clients/fides-js/src/components/Overlay.tsx

```diff
@@ -1,58 +1,51 @@
-import { h, FunctionComponent } from "preact";
```

Comment:
Overlay has gone through a big refactor to remove everything that made it PrivacyNotice dependent. Instead, `NoticeOverlay` controls all notice related logic, while `Overlay` is mostly just in charge of controlling showing banner/modal logic